bkmonitor/locale/en/LC_MESSAGES/django.po

```diff
@@ -11431,44 +11134,50 @@ msgstr "Lack of label_name"
 msgid "常用"
 msgstr "Commonly Used"
 
-msgid "依赖exceptionbeat采集器, 在节点管理安装"
-msgstr "Relying on the ExceptionBeat collector, install and install at BK-Nodeman"
+msgid "依赖bkmonitorbeat采集器, 在节点管理安装"
+msgstr "Dependent on bkmonitorbeat Collector, Install via Node Management"
 
 msgid "通过对挂载磁盘的文件状态ro进行判断，类似Linux命令：fgrep ' ro,' /proc/mounts"
 msgstr "Judging by the file status of the mount disk, similar to the Linux command: Fgrep 'Ro,' /Proc /Mounts"
 
 msgid "查看corefile生成路径：cat /proc/sys/kernel/core_pattern，确保在某一个目录下，例如 /data/corefile/core_%e_%t"
 msgstr "View Corefile generating path: Cat/Proc/SYS/KERNEL/CORE_PATTERN, ensure that in a certain directory, such as/data/corefile/core_%e_%t"
 
-msgid "依赖exceptionbeat采集器, 在节点管理安装,会自动根据core_pattern监听文件目录"
-msgstr "Dependence on ExceptionBeat collector, in BK-Nodeman and installation, it will automatically follow the Core_Pattern monitoring file directory"
+msgid "依赖bkmonitorbeat采集器, 在节点管理安装,会自动根据core_pattern监听文件目录"
+msgstr "Dependent on bkmonitorbeat Collector, Install via Node Management, Will Automatically Monitor File Directories Based on core_pattern"
```

Comment:
msgstr "Dependence on bkmonitorbeat collector, in BK-Nodeman and installation, it will automatically follow the Core_Pattern monitoring file directory"